keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/SCMInfoService.kt

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.keel.services
+
+import com.netflix.spinnaker.igor.ScmService
+import com.netflix.spinnaker.keel.api.SCMInfo
+import org.springframework.stereotype.Component
+
+@Component
+class SCMInfoService(
```

Comment:
What's the purpose of this component? Any reason you can't just wire in `ScmService` and have it extend the new `SCMInfo` interface you created?

Reply:
done